config/manager/manager.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    control-plane: starter-kit-operator
+  name: system
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: starter-kit-operator
+  namespace: system
+  labels:
+    control-plane: starter-kit-operator
+spec:
+  selector:
+    matchLabels:
+      control-plane: starter-kit-operator
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        control-plane: starter-kit-operator
+    spec:
+      containers:
+      - command:
+        - /manager
+        args:
+        - --leader-elect
+        image: jmeis/controller:latest
```

Comment:
Should this point to your name?

Reply:
I could revert this to what it was before (`controller:latest`), but that doesn't actually point to a real image. This image works, but I agree that I don't like how it's hardcoded. I left it like this though since it's easy to find out what images you would need to change if you want to run against your own local dev.

Reply:
We should make a note in the new issue for migrating the other makefile stuff to add some code to modify this automatically when running locally. I used a little yq for that.